app/page.tsx

```diff
@@ -162,7 +161,8 @@ export default async function Dashboard() {
 
       <div className="container mx-auto py-10">
         <MessageTable columns={experimentColumns} data={experimentAndBranchInfo} />
+        <a className="scroll-m-20 text-s" href="/api/auth/logout">Logout</a>
```

Comment:
How about some whitespace above this to visually separate it from the table a bit?

Reply:
I'm new to scroll margin and scroll snapping.  I'm curious what the intent here is -- I can't see any visual difference on my Mac if I disable it.

Reply:
I think I added the scroll margin to match the table before I decided to put the link inside the <div>. It doesn't seem to have an effect now, so I've removed it. Adding padding to the <a> tag also doesn't seem to be having an effect however? I'm going to keep playing with this but I don't think we should block the PR on it :)